### PR TITLE
Add Italian Javascript course

### PR DESCRIPTION
**Pull request by mcicolella**
## What does this PR do?
Add resource

### Description
Add Italian Javascript video course

## Checklist:
- [ x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [ x] Add needed indications (PDF, access notes, under construction).
- [ x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

**Comment by mcicolella** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Your link is broken

File: courses/free-courses-it.md [hunk 1]
```diff
@@ -129,6 +129,7 @@
 * [Corso base di Javascript](https://www.youtube.com/playlist?list=PLFLSwyN4GsWmcBvMr5tzsJy9TI8DbfWC5) - S. Tocci
 * [Corso di JavaScript](https://www.youtube.com/playlist?list=PLG5caACNVwzpIhlLACNZd6BvABWv_Ti4I) - Lacerba
 * [Corso Javascript (ES6)](https://www.youtube.com/playlist?list=PL0qAPtx8YtJceyk5_NpNvLbbkrmfX9kkw) - F. Camuso
+* [Corso Javascript Completo 2022](https://www.youtube.com/playlist?list=PLP5MAKLy8lP9FUx06-avV66mS8LXz7_Bb) - E. Midali
```
Comment (davorpa) [inline courses/free-courses-it.md:132]:
```suggestion
* [Corso Javascript Completo 2022](https://www.youtube.com/playlist?list=PLP5MAKLy8lP9FUx06-avV66mS8LXz7_Bb) - Edoardo Midali
```

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
welcome back @mcicolella !

**Pull request merged**